docs/docs/upgrading/migrate_v5_v6.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 1
+---
+
+# Upgrading to ICS v6.x from v5.x
+
+ICS specific changes are outlined below.
+
+Pre-requisite version for this upgrade: any from the `v5.x` release line.
+
+## Note
+
+## Provider
+
+### Migration (v5 -> v6)
+
+ConensusVersion was bumped to `6`.
+
+### Governance proposals
+
+To submit a proposal to add/modify/remove a consumer use the following command
+```shell
```

Comment:
I would recommend adding a tiny bit of info here, along the lines of:
`This was the command before: ```command```, now use this new command instead:`

btw, why is this in the migration? I think this should be in the changelog instead, because this doesn't actually have anything to do with migrating the state (which I think the upgrading file is usually)
Maybe this should be in an "integrators guide" like I did for the inactive vals here https://github.com/cosmos/interchain-security/pull/2133/files#diff-d3b7b6fffa7d0deffa451f04bcf224da77c989b9ada8c27598cf32817661cc8f